services/botVerificationService.js

```diff
@@ -10,6 +10,16 @@ const verifyToken = (token) => {
   return jwt.verify(token, config.get("botToken.botPublicKey"), { algorithms: ["RS256"] });
 };
 
+/**
+ * Verifies if the JWT for Discord Service is valid. Throws error in case of signature error or expiry
+ *
+ * @param token {String} - JWT to be verified
+ * @return {Object} - Decode value of JWT
+ */
+const verifyDiscordService = (token) => {
+  return jwt.verify(token, config.get("discordServiceToken.discordServicePublicKey"), { algorithms: ["RS256"] });
```

Comment:
```suggestion
  return jwt.verify(token, config.get("discordServiceToken.publicKey"), { algorithms: ["RS256"] });
```

Don't you think it was redundant?

Reply:
have updated it wrt to [this](https://github.com/Real-Dev-Squad/website-backend/pull/2413#discussion_r2082330355)